README.md

```diff
@@ -11,11 +11,18 @@ The source code is released under the MIT License.
     pip install python-rake
 
 ### Usage ###
+#for external .txt files
     import RAKE
-    Rake = RAKE.Rake([path_to_your_stopwords_file]);
-    # You can use one of the stoplists included in the repository under stoplists/
+    Rake = RAKE.Rake(<path_to_your_stopwords_file>); #take path as string datatype
+    Rake.run(text);
+#for lists
+    import RAKE
+    Rake = RAKE.Rake(<list>); #takes list input
+    Rake.run(text);
+#RAKE.SmartStopList() and RAKE.FoxStopList() return the expected lists as lists, they can be used as shown bellow
+    import RAKE
+    Rake = RAKE.Rake(RAKE.SmartStopList()); #takes list input
```

Comment:
Try the "view" link on the diff - https://github.com/justinkterry/python-rake/blob/d32efa9f758c6e39d5896cd6b2cc91406357b9b6/README.md You'll find that this section is mangled.

I think you want something closer to 

~~~
 ### Usage ###
for external .txt files

    import RAKE
    Rake = RAKE.Rake(<path_to_your_stopwords_file>); #take path as string datatype
    Rake.run(text);

for lists:

    import RAKE
    Rake = RAKE.Rake(<list>); #takes list input
    Rake.run(text);

`RAKE.SmartStopList()` and `RAKE.FoxStopList()` return the expected lists as lists, they can be used as shown bellow:

    import RAKE
    Rake = RAKE.Rake(RAKE.SmartStopList()); #takes list input
~~~